sherpa-onnx/csrc/online-recognizer-transducer-impl.h

```diff
@@ -183,6 +184,39 @@ class OnlineRecognizerTransducerImpl : public OnlineRecognizerImpl {
            s->NumFramesReady();
   }
 
+  void WarmpUpRecognizer(int32_t warm_up,int32_t max_batch_size) {
```

Comment:
```suggestion
  void WarmpUpRecognizer(int32_t warm_up,int32_t max_batch_size) const {
```

Also, please do an early return if `warm_up` is 0.

Reply:
Handled !